cmd/vegawallet/commands/service_config_describe.go

```diff
@@ -0,0 +1,87 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"code.vegaprotocol.io/vega/cmd/vegawallet/commands/cli"
+	"code.vegaprotocol.io/vega/cmd/vegawallet/commands/flags"
+	"code.vegaprotocol.io/vega/cmd/vegawallet/commands/printer"
+	"code.vegaprotocol.io/vega/paths"
+	"code.vegaprotocol.io/vega/wallet/service"
+	svcStoreV1 "code.vegaprotocol.io/vega/wallet/service/store/v1"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	describeServiceConfigLong = cli.LongDesc(`
+	    Describe the service configuration.
+	`)
```

Comment:
nit-pick, but why the trailing whitespace in the strings here?

Reply:
They stripped by `cli.LongDesc()`. It's easier to get the text right.